app/infrastructure/database/orms/course_orm.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'sequel'
+
+module HobbyCatcher
+  module Database
+    # Object-Relational Mapper for Courses
+    # class CourseOrm < Sequel::Model(:courses)\
+    # binding.pry
```

Comment:
Orm file might have some problem and try not to upload `binding.pry` debugging stuff to version control